parser/parser_test.go

```diff
@@ -22,7 +22,13 @@ ADAPTER adapter1
 LICENSE MIT
 PARAMETER param1 value1
 PARAMETER param2 value2
-TEMPLATE template1
+TEMPLATE """{{ if .System }}<|start_header_id|>system<|end_header_id|>
```

Comment:
Is it possible to add a test with trailing spaces?

Reply:
Yup, just added